.buildkite/pipelines/on_merge.yml

```diff
@@ -90,9 +92,13 @@ steps:
   - command: .buildkite/scripts/steps/lint_with_types.sh
     label: 'Linting (with types)'
     agents:
-      queue: c2-16
+      queue: n2-16-spot
```

Comment:
I noticed this one seems to only be using 8 threads, should be bump it to 16 or decrease to n2-8?

Reply:
Nope, the CPU load is currently about 14-16, so it should be good where it is.